tests/test_algorithm.py

```diff
@@ -285,6 +290,58 @@ def nop(*args, **kwargs):
 
         self.assertIs(composer, zipline.utils.events.ComposedRule.lazy_and)
 
+    def test_asset_lookup(self):
+        metadata = {0: {'symbol': 'PLAY',
+                        'asset_type': 'equity',
+                        'start_date': '2002-01-01',
+                        'end_date': '2004-01-01'},
+                    1: {'symbol': 'PLAY',
+                        'asset_type': 'equity',
+                        'start_date': '2005-01-01',
+                        'end_date': '2006-01-01'},
+                    2: {'symbol': 'OMG15',
+                        'asset_type': 'future'}}
+        algo = TradingAlgorithm(asset_metadata=metadata)
+
+        # Test before either PLAY existed
+        algo.datetime = pd.Timestamp('2001-12-01', tz='UTC')
+        self.assertEqual(2, algo.symbol('OMG15'))
+        with self.assertRaises(SymbolNotFound):
+            algo.symbol('PLAY')
+        with self.assertRaises(SymbolNotFound):
+            algo.symbols('PLAY', 'OMG15')
+
+        # Test when first PLAY exists
+        algo.datetime = pd.Timestamp('2002-12-01', tz='UTC')
+        self.assertEqual(2, algo.symbol('OMG15'))
+        self.assertEqual(0, algo.symbol('PLAY'))
+        list_result = algo.symbols('PLAY', 'OMG15')
```

Comment:
This check is failing for me because the returned list contains Assets, not just ints. Is there an easy way to convert the list, or should I just leave the original test?